src/collective/listmonk/listmonk.py

```diff
@@ -0,0 +1,28 @@
+from plone.restapi.testing import RelativeSession
+from typing import Optional
+from zExceptions import BadRequest
+
+
+# TODO get real values from configuration
+listmonk = RelativeSession("http://localhost:9000/api")
+listmonk.auth = ("admin", "admin")
+
+
+def call_listmonk(method, path, **kw):
```

Comment:
Add annotation for result:
```Union[dict, list]```

Reply:
tbh I don't think that's much better than leaving the type out entirely. A fully typed Python client for listmonk would have a separate function for each call, with a response type that has the correct fields. But I need this quickly, so I won't do that right now.